pom.xml

```diff
@@ -32,10 +32,10 @@
                     <source>1.8</source>
                     <target>1.8</target>
                     <!--<archive>-->
-                        <!--<manifest>-->
```

Comment:
Separate project (build to jar)
**Import it to maven as gal or plugin. mvn -PjilPrepare <-> mvn jilPrepare**
Properties config or form the main(prop, sours, out)
Clean code. :)